common/src/main/kotlin/entity/DiscordMessage.kt

```diff
@@ -309,7 +307,7 @@ enum class MessageFlag(val code: Int) {
 }
 
 @Serializable(with = MessageFlags.Serializer::class)
-class MessageFlags internal constructor(val code: Int) {
+data class MessageFlags internal constructor(val code: Int) {
```

Comment:
reason:
- data class to make `assertEquals` in ``MessageTest.`User serialization`()`` pass
- data class seems appropriate
- `equals`/`hashCode`: `code` is the only property that differentiates `MessageFlags` instances (`flags` is computed from `code`)